docs/troubleshooting/authentication/delete_user_action_is_not_working_properly.md

```diff
@@ -0,0 +1,25 @@
+---
+keywords: ['properly', 'delete', 'working']
```

Comment:
the keywords dont make sense here, can we add proper keywords that is related to the topic?